### ISSUE TITLE
Firebug do not working

### ISSUE THREAD

**Issue by beholdr** [bwinton/VerticalTabs#164]
Press F12 (or select «Inspect Element with Firebug») do nothing while this extension is active.
Firefox 46.0.1, firebug 2.0.16

**Comment by johngruen** [comment]
@beholdr can we get a screenshot of your browser and browser console (ctrl+shift-j)?

**Comment by beholdr** [comment]
Here it is. Console after reload page and press F12 (Firebug did not appear).
<img width="1398" alt="2016-05-12 9 20 15" src="https://cloud.githubusercontent.com/assets/741973/15205856/d8715bf2-182b-11e6-9cdb-43d51b4c996d.png">

**Comment by bwinton** [comment]
Hmmm.  There's nothing in the Browser Console…  Can I get a picture of what it looks like without Tab Center installed, for comparison?

**Comment by beholdr** [comment]
OK. First I unload all my tabs and launch FF without VerticalTabs (Firebug is working). Here is full console output from browser start:

<img width="1398" alt="2016-05-12 17 11 51" src="https://cloud.githubusercontent.com/assets/741973/15217748/fe8c5962-186d-11e6-9350-ca4aab6d3904.png">

Then I activated again VerticalTabs, close and open browser (Firebug didn't work). Here is full console output from browser start:

<img width="1398" alt="2016-05-12 17 13 30" src="https://cloud.githubusercontent.com/assets/741973/15217792/34e37c70-186e-11e6-9f3a-f494b53fade9.png">

Interesting that in menu Firebug shows as opened (checkmark is set), but it's not visible:

<img width="733" alt="2016-05-12 17 23 26" src="https://cloud.githubusercontent.com/assets/741973/15217972/e089255c-186e-11e6-8f1e-e21faa09393d.png">

Is it on your browser Firebug works fine with VerticalTabs?

**Comment by beholdr** [comment]
Oh, I tried to undock Firebug (float mode) and it's working with VerticalTabs:

<img width="1397" alt="2016-05-12 17 27 24" src="https://cloud.githubusercontent.com/assets/741973/15218102/6af461b6-186f-11e6-8379-7ff9ecc61f66.png">

So it's docking mode (attached to browser window) cause troubles.

**Comment by ericawright** [comment]
Can't reproduce, this *may* have been fixed with 24b4a5ec15a886f0943eea031f577f8f30490cb3

**Comment by bwinton** [comment]
Hmm, if you dock firebug to the left, it does look kinda odd…

**Comment by ericawright** [comment]
true, it doesn't shift - needs to be treated the same as bookmarks

**Comment by beholdr** [comment]
I've updated to v.1.18 and firebug now works fine. Thanks!

**Comment by bwinton** [comment]
I'm not sure how much we're going to cater to other add-ons…  Having said that, if we decide to do the work to be compatible, Firebug seems like one of the top priority ones, so I think I'm going to re-open this to track that work.

**Comment by Keith94** [comment]
According to [official website](http://getfirebug.com/) "The Firebug extension isn't being developed or maintained any longer."
